<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFle">
    <property name="title">
     <string>Fle</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuResources">
    <property name="title">
     <string>Resources</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
    <addaction name="actionAdd_list"/>
   </widget>
   <addaction name="menuFle"/>
   <addaction name="menuResources"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionAdd_list"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionAdd_list">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/add_list.png</normaloff>:/icons/add_list.png</iconset>
   </property>
   <property name="text">
    <string>Add list</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
